sherpa/astro/ui/utils.py

```diff
@@ -10431,13 +10431,78 @@ def get_fit_plot(self, id=None, recalc=True):
 
     get_fit_plot.__doc__ = sherpa.ui.utils.Session.get_fit_plot.__doc__
 
-    # copy doc string from sherpa.utils
     def get_model_component_plot(self, id, model=None, recalc=True):
+        """Return the data used to create the model-component plot.
+
+        .. versionchanged :: 4.13.2
+           For PHA data, the response model is now automatically added
+           by the routine unless the model contains a response.
```

Comment:
```suggestion
           by the routine unless the model contains a response already.
```

Reply:
I haven't made this change as (for me) it doesn't add anything.